openmdao/docs/openmdao_book/features/reports/reports_system.ipynb

```diff
@@ -103,7 +103,9 @@
     "\n",
     "1. The user has the option of setting a common output directory for ALL problems using the `OPENMDAO_WORKDIR` environment variable. Thus there is a single directory that contains all OpenMDAO output data.\n",
     "\n",
-    "2. The [openmdao clean](om-command-clean) or `openmdao.api.clean_outputs` function can be used to remove these directories.\n"
+    "2. The [openmdao clean](om-command-clean) or `openmdao.api.clean_outputs` function can be used to remove these directories.\n",
+    "\n",
+    "In addition to setting `OPENMDAO_WORKDIR` to set the top level output directory for all problems, a user can set `work_dir` as an argument or option of Problem in order to set the top level output directory under which that problems `'{problem_name}_out'` directory will be placed.\n"
```

Comment:
typo: "under which that problems"  -> "under which that problem's"

Reply:
"In addition to" should probably say "As an alternative to"...  

There is a question of precedence I guess.. it might be worthwhile to say that the problem option supersedes the env var...